docs/Acceptable-Casks.md

```diff
@@ -1,3 +1,7 @@
+---
+last_review_date: "1970-01-01"
```

Comment:
Can we hold off adding this review date until we've actually (re-)reviewed these documents?

Reply:
I thought this was a good "if it's 1970 then it needs review", and gets the metadata used. (We can make a list of docs that need re-review and put it in an issue.)

Reply:
I think ideally we'd go with "if it's blank" or "if it's false" or something: we need to review. The 1970 date is a little confusing and harder to "scan" than those.